src/Breakpoint.php

```diff
@@ -222,39 +205,46 @@ public function toGql(array $args): array
         $data = [
             'asset' => $this->asset,
             'label' => $this->label,
-            'value' => $this->value,
-            'unit' => $this->unit,
-            'ratio' => $this->ratio,
-            'mediaString' => $this->getMediaString(),
+            'minWidth' => $this->minWidth,
+            'widthUnit' => $this->widthUnit,
+            'sources' => $this->sources()->map(function (Source $source) use ($args) {
+                return $source->toGql($args);
+            })->all(),
+            // TODO: There is no neat way to separate placeholder string from srcset string,
+            // TODO: cause placeholder argument affects both.
             'placeholder' => $args['placeholder'] ? $this->placeholder() : null,
-            'srcSet' => $this->getSrcSet($args['placeholder']),
         ];
 
-        if ($args['webp']) {
-            $data['srcSetWebp'] = $this->getSrcSet($args['placeholder'], 'webp');
-        }
-
-        if ($args['avif']) {
-            $data['srcSetAvif'] = $this->getSrcSet($args['placeholder'], 'avif');
+        // Check if DimensionCalculator is instance of ResponsiveDimensionCalculator
+        // as ratio is only property applicable only for this DimensionCalculator
+        if (app(DimensionCalculator::class) instanceof ResponsiveDimensionCalculator) {
+            $data['ratio'] = app(DimensionCalculator::class)->breakpointRatio($this->asset, $this);
```

Comment:
For legacy purposes, `ratio` seems only relevant if you are using the original `ResponsiveDimensionCalculator`.